workers_tail.go

```diff
@@ -0,0 +1,114 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+var (
+	ErrMissingScriptName = errors.New("required script name missing")
+	ErrMissingTailID     = errors.New("required tail id missing")
+)
+
+type WorkersTail struct {
+	ID        string     `json:"id"`
+	URL       string     `json:"url"`
+	ExpiresAt *time.Time `json:"expires_at"`
+}
+
+type StartWorkersTailParameters struct {
+	AccountID  string
+	ScriptName string
+}
+
+type StartWorkersTailResponse struct {
+	Response
+	Result WorkersTail
+}
+
+type ListWorkersTailParameters struct {
+	AccountID  string
+	ScriptName string
+}
+
+type ListWorkersTailResponse struct {
+	Response
+	Result WorkersTail
+}
+type DeleteWorkersTailParameters struct {
+	AccountID  string
+	ScriptName string
+	TailID     string
+}
+
+// StartWorkersTail Starts a tail that receives logs and exception from a Worker
+//
+// API reference: https://api.cloudflare.com/#worker-tail-logs-start-tail
+func (api *API) StartWorkersTail(ctx context.Context, params StartWorkersTailParameters) (WorkersTail, error) {
```

Comment:
something like this is expected now 

```suggestion
func (api *API) StartWorkersTail(ctx context.Context, rc *ResourceContainer, scriptName string) (WorkersTail, error) {
```